api/src/routers/reservation.js

```diff
@@ -0,0 +1,69 @@
+import express from "express";
```

Comment:
You can see the code in `meal.js` and `reservation.js`, Try to reflect on whether its best to have the code repeated or merging the code into shared code.